salt/modules/chocolatey.py

```diff
@@ -369,7 +369,6 @@ def install(name,
             force_x86=False,
             package_args=None,
             allow_multiple=False,
-            no_progress=False,
```

Comment:
What about people who are using this presently?

Reply:
This is on Develop, so people shouldn't be using it. The option to disable the progress bar wasn't there in older versions of chocolatey. This only disables it on versions of chocolatey that support it.